packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -1534,4 +1640,4 @@ class _RenderAppBarTitleBox extends RenderAligningShiftedBox {
     size = constraints.constrain(child!.size);
     alignChild();
   }
-}
+}
```

Comment:
This should also be reverted. There should be a newline at the end of every file.